doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst

```diff
@@ -104,6 +104,412 @@ Thingy:91 has a multimode modem, which enables it to support automatic switching
+         When the update is complete, a "Completed successfully" message appears.
+      #. Scroll up in the menu on the left to the **FILE** section and click :guilabel:`Clear files`.
+
+   .. group-tab:: Through external debug probe
+
+      To update the firmware using the nRF9160 DK as the external debug probe, complete the following steps:
+
+      1. Open `nRF Connect for Desktop`_ and launch the Programmer app.
+
+      .. _prepare_hw_ext_dp:
+
+      2. Prepare the hardware:
+
+         a. Connect the Thingy:91 to the debug out port on a 10-pin external debug probe using a JTAG cable.
+
+            .. figure:: images/programmer_thingy91_connect_dk_swd_vddio.svg
+               :alt: Thingy:91 - Connecting external debug probe
+
+               Thingy:91 - Connecting external debug probe
+
+            .. note::
+               When using nRF9160 DK as the debug probe, make sure that VDD_IO (SW11) is set to 1.8 V on the nRF9160 DK.
+
+         #. Make sure that the Thingy:91 and the external debug probe are powered on.
+
+            .. note::
+               Do not unplug or power off the devices during this process.
+
+         #. Connect the external debug probe to the computer with a micro-USB cable.
+
+            In the Programmer navigation bar, :guilabel:`No devices available` changes to :guilabel:`SELECT DEVICE`.
+
+            .. figure:: ../nrf52/images/programmer_select_device1.png
+               :alt: Programmer - Select device
+
+               Programmer - SELECT DEVICE
+         #. Click :guilabel:`SELECT DEVICE` and select the appropriate debug probe entry from the drop-down list.
+
+            Select nRF9160 DK from the list.
+
+            .. figure:: images/programmer_com_ports.png
+               :alt: Programmer - nRF9160 DK
+
+               Programmer - nRF9160 DK
+
+            The button text changes to the SEGGER ID of the selected device, and the Device Memory Layout section indicates that the device is connected.
+
+      #. Set the SWD selection switch **SW2** to **nRF52** on the Thingy:91.
+         See `SWD Select`_ for more information on the switch.
+
+      #. In the menu on the left, click :guilabel:`Add file` in the **FILE** section, and select :guilabel:`Browse`.
+         A file explorer window appears.
+
+         .. figure:: images/programmer_add_file1.png
+            :alt: Programmer - Add file
+
+            Programmer - Add file
+
+      #. Navigate to where you extracted the firmware.
+
+      #. Open the folder :file:`img_app_bl` that contains the HEX files for flashing with a debugger.
+         See the :file:`CONTENTS.txt` file for information on which file you need.
+
+      #. Select the Connectivity bridge firmware file.
+      #. Click :guilabel:`Open`.
+      #. Scroll down in the menu on the left to the **DEVICE** section and click :guilabel:`Erase & write`.
+         The update is completed when the animation in Programmer's Device memory layout window ends.
```

Comment:
Bold since it's a UI element and the sentence is hard to parse otherwise.

Reply:
Fixed in a couple of places.